docs/admin/scale.md

```diff
@@ -17,43 +17,79 @@ Coder runs workspace operations in a queue. The number of concurrent builds will
 
 ## Infrastructure recommendations
 
+> Note: The below are guidelines for planning your infrastructure. Your mileage may vary depending on your templates, workflows, and users.
+
+When planning your infrastructure, we recommend you consider the following:
+
+1. CPU and memory requirements for `coderd`. We recommend allocating 1 CPU core and 2 GB RAM per `coderd` replica at minimum. See [Concurrent users](#concurrent-users) for more details.
+1. CPU and memory requirements for external provisioners, if applicable. We recommend allocating 1 CPU core and 1 GB RAM per 5 concurrent workspace builds to external provisioners. Note that this may vary depending on the template used. See [Concurrent workspace builds](#concurrent-workspace-builds) for more details.
```

Comment:
Can you link to the external provisioners docs here? Also,  does this mean we always recommend people separate their provisioners or can we provide some generic `coderd` sizing recommendations for coderd+n provisioners

Reply:
> Also, does this mean we always recommend people separate their provisioners or can we provide some generic coderd sizing recommendations for coderd+n provisioners

Fair. 0.25C/256MB per additional provisioner should be sufficient for most cases. I don't think separating provisioners is going to suit everybody.